### PR TITLE
Add more Benchmarks

### PR DESCRIPTION
**Pull request by tbosch**

### CODE REVIEW

File: npm-shrinkwrap.clean.json [hunk 1]
```diff
@@ -3333,6 +3333,9 @@
     "ieee754": {
       "version": "1.1.6"
     },
+    "incremental-dom": {
```
Comment (vicb) [inline npm-shrinkwrap.clean.json]: fix commit comment

File: modules/benchmarks/src/tree/ng2_static/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```
Comment (vicb) [inline modules/benchmarks/src/tree/ng2_static/index.html]: could you format this file ?

File: modules/benchmarks/src/tree/ng2_static/index.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    tree.data = buildTree();
+    appRef.tick();
+  }
+
+  function noop() {}
+
+  function init() {
+    enableProdMode();
+    platformBrowserDynamic().bootstrapModule(AppModule).then((ref) => {
+      var injector = ref.injector;
```
Comment (vicb) [inline modules/benchmarks/src/tree/ng2_static/index.ts]: const

File: modules/benchmarks/src/tree/ng2_static/tree.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import {Component, Input, NgModule} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+
+import {TreeNode, emptyTree, maxDepth} from '../util';
+
+function createTreeComponent(level: number, isLeaf: boolean) {
+  const nextTreeEl = `tree${level+1}`;
+  let template = isLeaf ?
```
Comment (vicb) [inline modules/benchmarks/src/tree/ng2_static/tree.ts]: const

File: modules/benchmarks/src/tree/baseline_dom/index.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```
Comment (vicb) [inline modules/benchmarks/src/tree/baseline_dom/index.html]: format

File: modules/benchmarks/src/tree/incremental_dom/index.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```
Comment (vicb) [inline modules/benchmarks/src/tree/incremental_dom/index.html]: format

**Comment by vicb** [comment]
Great !
Minor nits only.

File: modules/benchmarks/src/tree/ng2_static/index.ts (hunk 3 above)
Comment (tbosch) [inline modules/benchmarks/src/tree/ng2_static/index.ts]: Done

File: modules/benchmarks/src/tree/ng2_static/index.html (hunk 2 above)
Comment (tbosch) [inline modules/benchmarks/src/tree/ng2_static/index.html]: Done

File: modules/benchmarks/src/tree/ng2_static/tree.ts (hunk 4 above)
Comment (tbosch) [inline modules/benchmarks/src/tree/ng2_static/tree.ts]: Done

File: modules/benchmarks/src/tree/baseline_dom/index.html (hunk 5 above)
Comment (tbosch) [inline modules/benchmarks/src/tree/baseline_dom/index.html]: Done

File: modules/benchmarks/src/tree/incremental_dom/index.html (hunk 6 above)
Comment (tbosch) [inline modules/benchmarks/src/tree/incremental_dom/index.html]: Done

File: npm-shrinkwrap.clean.json (hunk 1 above)
Comment (tbosch) [inline npm-shrinkwrap.clean.json]: Done

**Comment by mprobst** [comment]
Merged as 2581c0851a257e834683f460443831877dba4d21.

**Pull request closed without merging**